source/reference/minio-mc/mc-support-diagnostics.rst

```diff
@@ -0,0 +1,132 @@
+==========================
+``mc support diagnostics``
+==========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support diagnostics
+
+.. mc:: mc support diag
+
+Description
+-----------
+
+You can check the health of your configuration of the underlying infrastructure that runs MinIO.
+
+The resulting health report is intended for use by MinIO Engineering via |subnet| and may contain internal or private data points, such as hostnames.
+Exercise caution before sending a health report to a third party or posting the health report in a public forum.
+
+MinIO recommends that you run the health diagnostics when first provisioning the cluster and again at any failure scenario.
+
+This diagnostic test runs as a one-shot test and can run for a long time.
+This test uses the majority of the available resources on the system. 
+Use care when running this diagnostic test to debug failure scenario to prevent larger outages. 
+
+Use the :mc-cmd:`mc support diagnostics` command to trigger the diagnostic test.
+You can then upload the results of the test can to the MinIO subscription network, |subnet|.
+
+
+Sample Output
+~~~~~~~~~~~~~
+
+.. code-block:: shell
+
+   ● Admin Info ... ✔ 
+   ● CPU ... ✔ 
+   ● Disk Hardware ... ✔ 
+   ● Os Info ... ✔ 
+   ● Mem Info ... ✔ 
+   ● Process Info ... ✔ 
+   ● Config ... ✔ 
+   ● Drive ... ✔ 
+   ● Net ... ✔ 
+   *********************************************************************************
+                                   WARNING!!
+        ** THIS FILE MAY CONTAIN SENSITIVE INFORMATION ABOUT YOUR ENVIRONMENT ** 
+        ** PLEASE INSPECT CONTENTS BEFORE SHARING IT ON ANY PUBLIC FORUM **
+   *********************************************************************************
+   mc: Health data saved to dc-11-health_20220511053323.json.gz
+
+The gzipped output contains the requested health information.
+
+
+Examples
+--------
+
+Upload Health Data for a Cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Generate health data for a MinIO cluster at alias ``minio1`` for transmission to |SUBNET|.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support diag minio1
+
```

Comment:
Should note the auto-upload only works for clusters registered w/ SUBNET .e.g using `mc support register`